styles/src/shadycss/shared-styles-host.ts

```diff
@@ -50,11 +50,16 @@ export class ShadyCSSSharedStylesHost extends DomSharedStylesHost {
     processStylesheets(this.usingApply);
   }
 }
-
+/**
+ * Factory to resolve runtime errors for Ivy compilation
```

Comment:
Nit: let's change the documentation here to just mention it's a factory instead of Ivy, since factories are a common pattern in Angular.

```suggestion
 * Factory that creates a new ShadyCSSSharedStylesHost.
```

Reply:
Thanks for the suggestion @asyncLiz . I have committed the suggestion. Please test it on Monday and merge it.